.github/ISSUE_TEMPLATE/story_template.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Story Template
+about: Story template for tracking on the infra project board 
```

Comment:
I'm not sure everyone will immediately know what "Story" means in this context.

At the very least I think this description should make that more clear. Maybe it's even worth making multiple more direct templates. Like "New AWS Resource Request", "Maintenance Request", "Project Story".

Reply:
Will change to project story. This template was meant to map to our internal infra tasks. Then going to make a separate one for resource requests